### ISSUE TITLE
Failed to convert LSTM cell with activation and dropout settings

### ISSUE THREAD

**Issue by Andreas-Krebbel** [onnx/tensorflow-onnx#2002]
**Describe the bug**
Keras LSTM model setting activation function and dropout is decomposed instead of using ONNX LSTM cell.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Fedora 35, x86
- Tensorflow Version: 2.9.1
- Python version: 3.9.13
- tf2onnx: e896723 (2022-07-15)

**To Reproduce**
```
from keras.layers import Input, LSTM
from keras.models import Sequential
import tf2onnx

model = Sequential()
model.add(Input(shape=(16, 16)))
model.add(LSTM(16, activation='sigmoid',  dropout=0.1))

model.save('test.h5')
onnx_model = tf2onnx.convert.from_keras(model, opset=16, output_path='test.onnx')
```
Removing either activation='sigmoid' or dropout=0.1 generates an LSTM cell in the output as expected.

**Screenshots**
test.h5:
![Screenshot from 2022-07-18 11-33-49](https://user-images.githubusercontent.com/38103320/179484101-dd04144f-eb05-4ebd-920f-934fd33ca011.png)
test.onnx
![Screenshot from 2022-07-18 11-34-06](https://user-images.githubusercontent.com/38103320/179484114-ca7ce693-a599-40be-a2f5-b815c3dc1593.png)

**Comment by hwangdeyu** [comment]
tf2onnx is not expected supporting converting models for training so far, such as setting dropout.
But in this case, it seems like the root cause is from the `tensorflow OptimizeGraph`. We will consider to add `dependency` to optimization.
